packages/sync/src/modules/class-terms.php

```diff
@@ -319,4 +290,19 @@ public function expand_term_taxonomy_id( $args ) {
 	public function expand_terms_for_relationship( $relationship ) {
 		return get_term_by( 'term_taxonomy_id', $relationship->term_taxonomy_id );
 	}
+
+	/**
+	 * Return Total number of objects.
+	 *
+	 * @param array $config Full Sync config.
+	 *
+	 * @return int total
+	 */
+	public function total( $config ) {
+		global $wpdb;
+		$where = $this->get_where_sql( $config );
+		// phpcs:ignore WordPress.DB.PreparedSQL.InterpolatedNotPrepared
+		return $wpdb->get_var( "SELECT COUNT(*) FROM $wpdb->term_taxonomy WHERE $where" );
```

Comment:
Any reason why we're not using the default method? Since the table name is now `term_taxonomy` I think we should be able to use the method from the parent class.

Reply:
let's try that! good 🦅 👁 ! Thanks!

Reply:
https://github.com/Automattic/jetpack/pull/13963/commits/d7fec62560a8a91553a53667bcf182434433a155